frontend/src/cards/TableCard.tsx

```diff
@@ -83,13 +96,28 @@ export function TableCard(props: TableCardProps) {
       }
     >
       {([queryResponse]) => {
-        const dataWithoutUnknowns = queryResponse.data.filter(
+        let dataWithoutUnknowns = queryResponse.data.filter(
           (row: Row) =>
             row[props.breakdownVar] !== UNKNOWN &&
             row[props.breakdownVar] !== UNKNOWN_RACE &&
             row[props.breakdownVar] !== UNKNOWN_ETHNICITY
         );
 
+        if (showAltPopCompare(props)) {
+          // This should only happen in the vaccine kff state case
+          dataWithoutUnknowns = dataWithoutUnknowns.map((item) => {
+            return {
```

Comment:
i think we can do this to avoid having to redeclare all of the other properties that already exist on `item`; i'd have to test this out though not positive. 
```
const { vaccine_population_pct, acs_vaccine_population_pct, ...restOfItem } = item
return {  
              vaccine_population_pct: vaccine_population_pct || acs_vaccine_population_pct,
                ...restOfItem
            };
```

Reply:
Sorry just saw this, yeah that would be better let me try it out

Reply:
@benhammondmusic That totally works, way more elegant, thanks :)